### ISSUE TITLE
All form fields are on the first page

### ISSUE THREAD

**Issue by krispyk84** [derekblair/ILPDFKit#58]
Hi, 

When I put in a multi page form with fields set in Adobe Acrobat DC, all the fields are on page 1 of 5 and there are no fields on pages 2-5. Any idea why this happens and how I can fix it?

**Comment by AhmedShammala** [comment]
@krispyk84  thanks u for speed respond , did u use any other good alternative  ?

**Comment by krispyk84** [comment]
No, I needed it for a prototype app and the feature wasn’t critical so I gave up on it.
